src/models/real-fevr-opener.ts

```diff
@@ -23,11 +22,10 @@ export class OpenerRealFvr extends Model<OpenerRealFvrMethods> implements Deploy
       await super.loadContract();
 
     const purchaseToken = await this._purchaseToken() || this.purchaseTokenAddress;
-    if (!purchaseToken)
-      throw new Error(Errors.MissingERC20AddressOnContractPleaseSetPurchaseToken);
-
-    this._erc20 = new ERC20(this.web3Connection, purchaseToken);
-    await this._erc20.loadContract();
+    if (purchaseToken && purchaseToken !== '0x0000000000000000000000000000000000000000') {
```

Comment:
export a const with this value so we can _know_ what it is and import it for usage whenever possible

Reply:
done